tools/PI/DevHome.PI/Models/WERDisplayInfo.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace DevHome.PI.Models;
+
+public partial class WERDisplayInfo : ObservableObject
+{
+    public WERReport Report { get; }
+
+    [ObservableProperty]
+    private string _failureBucket;
+
+    [ObservableProperty]
+    private string _analyzeResults;
+
+    public WERDisplayInfo(WERReport report)
+    {
+        Report = report;
+        AnalyzeResults = InitializeAnalyzeResults();
+        FailureBucket = UpdateFailureBucket();
+        Report.PropertyChanged += Report_PropertyChanged;
+    }
+
+    private void Report_PropertyChanged(object? sender, PropertyChangedEventArgs e)
+    {
+        if (e.PropertyName == nameof(WERReport.CrashDumpPath))
+        {
+            AnalyzeResults = InitializeAnalyzeResults();
+            FailureBucket = UpdateFailureBucket();
+        }
+        else if (e.PropertyName == nameof(WERReport.FailureBucket))
+        {
+            FailureBucket = UpdateFailureBucket();
+        }
+    }
+
+    private string UpdateFailureBucket()
+    {
+        if (AnalyzeResults == string.Empty)
+        {
+            return Report.FailureBucket;
+        }
+
+        string[] lines = AnalyzeResults.Split('\n', StringSplitOptions.RemoveEmptyEntries);
+
+        foreach (string line in lines)
+        {
+            if (line.Contains("FAILURE_BUCKET_ID:"))
```

Comment:
I assume these things are never localized, right?

Reply:
Ugh... this was leftover from when I was running debugger extensions to create better crash buckets. I need to spend more time to make this extensible, so I removed the code for now.

I would suspect this wouldn't be localized. I got feedback that I should be consuming the XML-formatted output of !analyze in the future, and I suspect XML tags are not localized.